lib/render_template.rb

```diff
@@ -11,10 +11,32 @@ def initialize(template, locals: {})
   end
 
   def call
-    bind = binding
-    locals.each { |k, v| bind.local_variable_set(k, v) }
-
     vcl_file = File.join(File.dirname(__FILE__), "..", "vcl_templates", "#{template}.vcl.erb")
-    ERB.new(File.read(vcl_file), trim_mode: "-").result(bind)
+    ERB.new(File.read(vcl_file), trim_mode: "-").result(binding_with_local_variables(locals))
+  end
+
+  def render_partial(partial, indentation: "", locals: {})
+    partial_path = File.join(File.dirname(__FILE__), "..", "vcl_templates", "_#{partial}.vcl.erb")
+
+    output = ERB
+      .new(File.read(partial_path), trim_mode: "-", eoutvar: "_#{partial}_eoutvar")
```

Comment:
I find this line (and the similar one elsewhere) is quite hard to read. Is it worth splitting it up?